From a55a722cb059fa8439ac622a728e02a073b11a2a Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Mon, 6 Feb 2023 10:14:18 +0800
Subject: [PATCH 08/45] Revert "drm/msm/dp: do not complete
 dp_aux_cmd_fifo_tx() if irq is not for aux transfer"

This reverts commit 785607e5e6fb52caf141e4580de40405565f04f1.
---
 drivers/gpu/drm/msm/dp/dp_aux.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/msm/dp/dp_aux.c b/drivers/gpu/drm/msm/dp/dp_aux.c
index a73a2b28a70a6..19b35ae3e9272 100644
--- a/drivers/gpu/drm/msm/dp/dp_aux.c
+++ b/drivers/gpu/drm/msm/dp/dp_aux.c
@@ -423,10 +423,6 @@ void dp_aux_isr(struct drm_dp_aux *dp_aux)
 
 	aux->isr = dp_catalog_aux_get_irq(aux->catalog);
 
-	/* no interrupts pending, return immediately */
-	if (!aux->isr)
-		return;
-
 	if (!aux->cmd_busy)
 		return;
 
-- 
2.25.1

